source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -94,12 +94,18 @@ To set up the metrics exporter app:
 
 4. Run `cf start metric-exporter` to start your app
 1. Check you're generating Prometheus metrics at the metrics endpoint
-    - `https://<your-app-url>/metrics`
+    - `https://metric-exporter.cloudapps.digital/metrics`
 5. Bind your app to the Prometheus service
-  - `cf bind-service <app-name> <service-instance-name>`
+  - `cf bind-service metric-exporter <service-instance-name>`
+6. Map a route to your app so Prometheus can scrape it
+  - `cf map-route metric-exporter cloudapps.digital --hostname metric-exporter`
```

Comment:
this command will fail because the name `metric-exporter.cloudapps.digital` is taken I think. in any case, it'll only work once.

maybe `cf map-route metric-exporter cloudapps.digital --hostname <app-name>`?